config/manager/manager.yaml

```diff
@@ -38,6 +38,15 @@ spec:
         image: controller:latest
         args:
         - --leader-elect
+        env:
```

Comment:
It would be good to document how versions should be updated

Reply:
I've added a section to `CONTRIBUTING.md`.